plex/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_PLEX_IP
+      APP_PORT: $APP_PLEX_PORT
+
+  server:
+    image: ghcr.io/linuxserver/plex:version-1.28.1.6104-788f82488
+    user: "1000:1000"
+    restart: on-failure
+    environment:
+      - PUID=1001
+      - PGID=100
+      - VERSION=docker
+    volumes:
+      - ${APP_LIGHTNING_NODE_DATA_DIR}/downloads:/downloads
```

Comment:
This is most likely not the correct volume you want to mount. This should probably look like this:
```
 - ${APP_DATA_DIR}/downloads:/downloads
```

Reply:
Umbrel convention would actually be `- ${APP_DATA_DIR}/data:/downloads` but 🤷🏻

Reply:
Ah, I intended to point it to simple-torrent's data_dir. But to your point people might want to use plex without simple torrent data since plex does have other features